samples/drivers/led_pwm/boards/nucleo_f091rc.overlay

```diff
@@ -12,3 +21,7 @@
 &pwm2 {
 	status = "okay";
 };
+
+&spi1 {
+	status = "okay";
```

Comment:
In commit message, you say that spi1 is disabled to avoid a conflict. But here it is enabled :)

Reply:
right, sorry for this copy/paste error